### DISCUSSION TITLE
Waybackurls,figlets,httpx not found

### DISCUSSION THREAD

**Discussion by DestinyAnyaele** [blackhatethicalhacking/TerminatorZ#5] [Q&A] [answered]
This is a nice project but I have problem installing the requirements

After cloning the project in home directory Kali Linux, i didn't have go installed for installing the requirements so i installed it in home directory(The go was installed as a folder with lots of other folders), then I installed the requirements through the link in the home directory(same directory as go)

So while running the shell I received
figlet : command not found
Httpx : command not found
Waybackurls : command not found

I have tried to see what's wrong by reinstalling the requirements but still same error. I need help  on installing the requirements :'(

Or does the requirements have to be installed in the cloned project directory

**Comment by blackhatethicalhacking** [comment, marked as answer]
Hello!

Already answered here:

https://github.com/blackhatethicalhacking/TerminatorZ/issues/4

Install the tools that are first required, test them separately making sure they work, then run TerminatorZ.
